.flake8

```diff
@@ -1,7 +1,7 @@
 [flake8]
 extend-ignore = E203, E266, E501
 # line length is intentionally set to 80 here because black uses Bugbear
-# See https://github.com/psf/black/blob/master/docs/the_black_code_style.md#line-length for more details
+# See https://github.com/psf/black/blob/main/docs/the_black_code_style/current_style.md#line-length for more details
```

Comment:
I'd rather move references to documentation to ReadTheDocs. We're moving away from GitHub as a documentation host to only RTD because it's makes our lives easier creating good docs. I know this link is dead right now, but it won't be once we release a new version with the documentation reorganization changes (which should be soon!).

```suggestion
# See https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length for more details
```